ref/anbox-https-api.md

```diff
@@ -491,3 +494,211 @@ Platform | Field name       | Available since   | JSON type | Access | Descripti
+
+To get the list of available properties and areas, query first the [`1.0/vhal/config` endpoint](#heading--10vhalconfig).
+
+Example return value for `curl -s -X GET --unix-socket /run/user/1000/anbox/sockets/api.unix s/1.0/vhal/0x15600503/0x31 | jq .`:
+
+```bash
+{
+  "metadata": {
+    "value": {
+      "area_id": 49,         # Requested area.
+      "bytes": [],           # Raw bytes value as an 8-bit unsigned integer array.
+      "float_values": [      # Float array
+          16.0
+      ],
+      "int32_values": [],    # 32-bits signed integer array
+      "int64_values": [],    # 64-bits signed integer array
+      "prop": 358614275,     # Requested property.
+      "status": 0,           # Status of the property, see below.
+      "string_value": "",    # UTF-8 string value.
+      "timestamp": 0         # Time when the property was last set in nanoseconds since boot.
+    }
+  },
+  "status": "Success",
+  "status_code": 200,
+  "type": "sync"
+}
+```
+
+Usually, only one of `bytes`, `float_values`, `int32_values`, `int64_values`,
+`string_value` is set, and the rest is empty or omitted, depending on the
+property type (see [`1.0/vhal/config`](#heading--10vhalconfig)).
+`MIXED` property types may have multiple of these values set at the same time, see
+[VHAL property types](https://source.android.com/docs/automotive/vhal/property-configuration#property-types).
+
+Status can be one of the following values, taken from the
+[VhalPropertyStatus](https://cs.android.com/android/platform/superproject/+/android10-release:hardware/interfaces/automotive/vehicle/2.0/types.hal;l=2700-2720)
+enumeration:
+
+|Name|Value|Description|
+|-|-|-|
+|Available|0|Property is available and behaving normally|
+|Unavailable|1|Property is not available for reading or writing. Transient state|
+|Error|2|Property has an error and is not available.|
+
+
+#### PUT
+ * Description: Set a VHAL property to a new value
+ * Operation: sync
+ * Return: standard return value or standard error
+
+Example input:
+
+```bash
+{
+  "prop_id": 286261505,  # Property identifier.
+  "area_id": 0,          # Area identifier. For global properties, it should be set to 0.
+  "bytes": [],           # (Optional) Raw bytes as 8-bit unsigned integer array
+  "float_values": [],    # (Optional) Float array
+  "int32_values": [],    # (Optional) 32-bit signed integer array
+  "int64_values": [],    # (Optional) 64-bit signed integer array
+  "string_value": "Foo", # (Optioanl) UTF-8 string
+}
+```
+
+At least one of `bytes`, `float_values`, `int32_values`, `int64_values`,
+`string_value` must be set, or the query will be considered invalid.
+
+JSON does not allow for hexadecimal or octal integers, all integers (including
+`prop_id` and `area_id`) must be decimal.
+
+Return value for the input above:
+
+```bash
+{
+    "status": "Success",
+    "status_code": 200,
+    "type": "sync"
+}
+```
+
+<a name="heading--10vhalconfig"></a>
```

Comment:
These HTML anchors are no longer necessary. The new layout has a page level TOC which is auto-generated. See https://anbox-cloud.io/docs/exp/production for example.

It's best to leave out these anchors as I have a clean up task for them anyway. It breaks headings and navigation in RTD.